Lottery.java

```diff
@@ -0,0 +1,14 @@
+import java.util.Random;
+
+public class Lottery {
+    public static Ball getRandomBall(){
+        Random random = new Random();
+        int numb = random.nextInt(100);
+        ColorSupplier ColorsSupplier = new ColorSupplier();

```

Comment:
The variable name `ColorsSupplier` should follow camelCase naming conventions. Consider renaming it to `colorSupplier` for consistency with Java naming standards.